### ISSUE TITLE
Mobile: Custom Widget For Course Preview

### ISSUE THREAD

**Issue by ezgy** [bounswe/bounswe2022group2#408]
### Issue Description

The mobile team plans to start implementing the home screen. According to our design, we will have course previews -(with image and title) that are clickable to navigate to that course. Since this is a specific design, a custom widget should be implemented. 

### Step Details

Steps that will be performed:
- [x] Design a clickable widget with image and text
- [x] Make image and text variable

### Final Actions

Once this widget is complete, it will be shared with the mobile team. Then, it will be connected with a backend endpoint and tested. Finally a pr will be created to merge into dev branch.

### Deadline of the Issue

29.10.2022 23.59

### Reviewer

Egemen Atik

### Deadline for the Review

30.10.2022

**Comment by ezgy** [comment]
#430 is opened and ready to be merged once mobile team (@bahricanyesil @egemenatikk @onurkomurcu ) reviews.

**Comment by ezgy** [comment]
Here is a glimpse of the widget as default and with tapping animation active:
![default](https://user-images.githubusercontent.com/56553280/198836409-b6918a20-9bf0-4779-96b8-8ee34153e501.jpeg)
![tapping](https://user-images.githubusercontent.com/56553280/198836408-a80cbd68-1749-487e-9791-fa032d6bdea2.jpeg)

**Comment by ezgy** [comment]
pr #430 is merged and this task is completed.
